src/universal/redux/getAuthedUser.js

```diff
@@ -40,21 +40,13 @@ query {
 }`;
 
 const updateTokenMutationHandlers = {
-  updateUserWithAuthToken(optimisticVariables, queryResponse, currentResponse) {
+  acceptInvitation(optimisticVariables, queryResponse, currentResponse) {
```

Comment:
@mattkrick you ok with locating this mutation handler here? I can't anticipate we'd add too many more to this spot...